BLSQ/iaso
# IA-2043: Create a new permission - limit data sources access to view only

Explain what problem this PR is resolving
- Before every user has access to read and write data source permissions
- The changes will allow to restrict some user to read only permissions

Related JIRA tickets : [IA-2043](https://bluesquare.atlassian.net/browse/IA-2043)

## Self proofreading checklist


## Changes
- Added iaso_write_sources permission
- Added a button to see data source details
- Created a page to display data source details
- Let some user to be able to edit a data source and other to only read the data source details

## How to test
- Create or edit users add iaso_write_sources permissions and for other remove it
- Go to data source list
- Check which operations accessible to a user

## Print screen / video
![Peek 2023-07-27 18-57](https://github.com/BLSQ/iaso/assets/19631540/725000d9-23ac-410d-8507-4e8a29f2a2b4)


[IA-2043]: https://bluesquare.atlassian.net/browse/IA-2043?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
